src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -55,7 +55,7 @@ def _get_default_install_location(exe_name):
 
 name_arg_type = CliArgumentType(options_list=('--name', '-n'), metavar='NAME')
 
-register_cli_argument('acs', 'name', arg_type=name_arg_type, help='ACS cluster name', completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
+register_cli_argument('acs', 'name', arg_type=name_arg_type, configured_default='acs', help="ACS cluster name. You can configure the default using 'az configure --defaults acs=<name>'", completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
```

Comment:
I assume this file has disabled the pylint `line-too-long` rule, nevertheless, this is line is too long. Please break it to improve the readability.